application.js

```diff
@@ -11,9 +11,6 @@ securableEndpoints = [];
 
 var app = express();
 
-// Enable CORS for all requests
-app.use(cors());
```

Comment:
Body Parser, yes. `cors()` I think we're gonna need - sure this caused it?
We might need to find another way not using .pipe if so. Eventually, we'll be looking up request definitions in a DB anyway, so may not be streaming..